### PR TITLE
Added support for qlot

### PR DESCRIPTION
**Pull request by Gavinok**
Added basic support for qlot. Left it open ended enough that we can support different lisps in the long run. I decided to add a makefile to avoid the headache of people running things in the wrong directory. I am open to dropping it if you feel make is an unnecessary dependency.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+project = lisp
+
+all:run
+
+# If dependancies are not installed attempt to install them using
```
Comment (fstamour) [inline Makefile]: typo: `dependancies` → `dependencies`

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+project = lisp
```
Comment (fstamour) [inline Makefile]: What's this variable for?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+all:run
```
Comment (fstamour) [inline Makefile]: tiny style-warning: `all:run` → `all: run`

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+# If dependancies are not installed attempt to install them using
+# the qlot executable
+.qlot/setup.lisp:
+	command -v qlot >/dev/null && qlot install
+
+run: .qlot/setup.lisp
```
Comment (fstamour) [inline Makefile:9]: the `run` target should be marked as phony. if you create a file named `run` the makefile will not run this command if the file `run` is newer than the dependencies.

Just add the line `.PHONY: run`

File: README.md [hunk 5]
```diff
@@ -1,2 +1,38 @@
...
+Simply start up a repl at the root directory then in the repl run
+
+```lisp
+(ql:quickload :qlot)
+(qlot:install :non-trivial-benchmarks)
+```
+
+## Running the benchmark
+
+If the dependancies where installed correctly you can run the
```
Comment (fstamour) [inline README.md]: same typo `dependancies` -> `dependencies`

Comment (fstamour) [reply README.md]: oh, and it's `were` not `where`

**Review by fstamour — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (Gavinok) [reply Makefile]: Oh I meant to remove that. Thanks for pointing it out.

**Comment by fstamour** [comment]
Looks good, I'll merge after I've tested it, which I won't be able to do for about another 9-ish hours.

**Review by fstamour — APPROVED** [review]

**Comment by fstamour** [comment]
I'm getting this error when qlot tries to fetch anything...
```
Unable to load any of the alternatives:
   ("libcrypto.so.1.1" "libcrypto.so.1.0.0" "libcrypto.so")
```

This is an issue with qlot itself, not your PR. Also, I get this problem on NixOS, I'll try on another OS to see...

**Comment by fstamour** [comment]
On windows:
`(qlot:install ...)`  worked


but calling `make` failed, because there's a space in my username which ended up in my home directory, where quicklisp's local-projects is.... :

```
$ make
./scripts/run.sh
bash: C:\Users\Francis: No such file or directory
make: *** [Makefile:10: run] Error 127
```

**Comment by fstamour** [comment]
works like a charm on FreeBSD :shrug:

**Pull request merged**